EZAlertController.swift

```diff
@@ -43,17 +43,20 @@ open class EZAlertController {
     //==========================================================================================================
     // MARK: - Class Functions
     //==========================================================================================================
-
+    
+    @discardableResult
     open class func alert(_ title: String) -> UIAlertController {
```

Comment:
Does `@discardableResult` work on separate lines from the function print or does it need to be one line like

```
@discardableResult open class func alert(_ title: String) -> UIAlertController {
```